examples/layers/ios/.gitignore

```diff
@@ -0,0 +1,37 @@
+.idea/
```

Comment:
I think it's fine if we check this in as-is and then redo it later. I mean, this work is done, so...